pkg/core/workflow.go

```diff
@@ -97,49 +103,12 @@ const (
 	TaskNode NodeType = "TaskNode"
 )
 
-// NodeState represents a node in different stage.
```

Comment:
Umm why are these statuses deleted?

Reply:
We have use Conditions to describe that in kubernetes API, I think we have no necessary to translate it to these NodeState(s).